### ISSUE TITLE
[CaveControl] Minecraft seems to crash whenever I enter the Mining Dimension.

### ISSUE THREAD

**Issue by ghost** [ReikaKalseki/Reika_Mods_Issues#2118]
Whenever i'm exiting or entering Aroma's Mining Dimension, it crashes. Checking from the crash report, It seems to be CaveControl having a part in this. Can you confirm?

Crash report is here: https://pastebin.com/H4ZWn4Y5

Thanks!

**Comment by ReikaKalseki** [comment]
Only that dimension?

**Comment by ghost** [comment]
Yes. The nether perfectly fine (although it does lag-spike sometimes, but i'm 100% sure it's not CaveControl's fault), and The End seems to run smooth.

**Comment by ReikaKalseki** [comment]
What about the overworld, the only other dimension with Mineshafts?

**Comment by ghost** [comment]
The overworld almost never crashed - The only time it crashed was when i was traveling very fast into unloaded chunks. Not related  though.

**Comment by ReikaKalseki** [comment]
I wonder if this is related to #275...

**Comment by ghost** [comment]
Well, to be PERFECTLY honest, I don't have a machine. I can list the mods that are used in the modpack. (I did contact the modpack owner first, he told me that CaveControl is doing this and he doesn't know how to fix it. He linked me to this issue thread.)

**Comment by ghost** [comment]
And oh, by the way, the mods that belong to you in this modpack are CaveControl, DragonAPI and Chromaticraft. No Rotarycraft.

**Comment by Barhandar** [comment]
Won't be surprised if this is actually #2250 - that is, Aroma's mining _biome_ has incorrect data, but without _both_ CaveControl and CodeChickenLib the code that would crash on said invalid data is never reached; and mineshafts are only related because that's where code that fails to check for data validity is, similarly to strongholdgen.
